Makefile

```diff
@@ -30,7 +34,8 @@ docs: .tox
 
 release:
 	git fetch --tags
-	$(PY) setup.py sdist upload
+	$(PY) setup.py sdist
+	$(BIN)/twine upload --repository-url https://upload.pypi.org/legacy/ dist/*
```

Comment:
Do we need to clear old build artifacts from `dist/` so they don't get picked up here, or does twine handle that automatically?